frontend/src/metabase/visualizations/click-actions/lib/modes.ts

```diff
@@ -2,7 +2,12 @@ import type Question from "metabase-lib/v1/Question";
 
 import { Mode } from "../Mode";
 import { DefaultMode } from "../modes/DefaultMode";
+import { EmbeddingSdkMode } from "../modes/EmbeddingSdkMode";
 
 export function getMode(question: Question): Mode | null {
   return new Mode(question, DefaultMode);
 }
+
+export function getEmbeddingMode(question: Question): Mode | null {
```

Comment:
Does this function go away after we've implemented every drill in `EmbeddingSdkMode`? This is similar to my comment in `frontend/src/metabase/visualizations/click-actions/Mode/Mode.ts`